mycqu/card.py

```diff
@@ -201,5 +289,37 @@ def get_fee_data(synjones_auth, room, fee_item_id):
     if dic['msg'] == 'success':
         return dic
     else:
-        raise FeeAcquisitionFailed(dic['msg'])
+        raise AcquisitionFailed(dic['msg'])
+
+
+# 获取校园卡账号信息
+def get_card_data(session):
+    url = "http://card.cqu.edu.cn/NcAccType/GetCurrentAccountList"
+
+    res = session.post(url)
+    result = json.loads(json.loads(res.text))
+    if result['respCode'] != "0000":
+        raise AcquisitionFailed(error_msg=result['respInfo'])
+
+    return result['objs']
```

Comment:
感觉按照以往的命名风格这个更像 `get_card_raw`?
```suggestion
def get_card_raw(session):
    url = "http://card.cqu.edu.cn/NcAccType/GetCurrentAccountList"

    res = session.post(url)
    result = json.loads(json.loads(res.text))
    if result['respCode'] != "0000":
        raise AcquisitionFailed(error_msg=result['respInfo'])

    return result['objs']
```

Reply:
对这两个函数命名的问题，因为我之前考虑只返回card对象，card中含有List[bill]，所以用统一的get_card_raw方法来获取card生成所需的所有数据，将现在这两个函数命名为get_xxx_data，如果按照我在card和bill对象提出的新方案，这两个函数就应当命名为get_xxx_raw